Jamulus.pro

```diff
@@ -141,11 +141,23 @@ win32 {
     HEADERS += mac/activity.h
     OBJECTIVE_SOURCES += mac/activity.mm
     CONFIG += x86
-    QMAKE_TARGET_BUNDLE_PREFIX = net.sourceforge.llcon
+    QMAKE_TARGET_BUNDLE_PREFIX = io.jamulus
     QMAKE_APPLICATION_BUNDLE_NAME. = $$TARGET
 
+    QMAKE_PROVISIONING_PROFILE = 25e71953-9b1b-457a-a171-d63458583e7e
+    QMAKE_DEVELOPMENT_TEAM = ENJ2T4A3FZ
```

Comment:
For the non-Mac people, can you clarify where this comes from?
If this is non-obvious to others, it may even make sense to add this as a permanent comment to the code?

Reply:
Of course! 
From what I can tell, when QT's Mac-deploy script does the signing, it uses these keys to help. 

The provisioning profile is part of apple's code signing which essentially defines what rights the software has. The provisioning profile is something that is used in the store to tie together certificates and rights. The UUID is the reference to that.

The development team is the ID of the Bolton Technology consulting entity in the App Store - essentially the creator of the provisioning profile and certificates used to sign the build.

Now, I can try removing these and see if signing works fine - they may just be to populate the Xcode project.